lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -395,6 +395,13 @@ def add_order_source(doc, payment_method, options)
         end
       end
 
+      def google_token_type(payment_method)
```

Comment:
💬  Comment:

mmm maybe could be just a matter of taste but why have a method for something that could be easily been addressed by a `[:google_pay, :android_pay].incluide?(payment_method.source)` ?

Reply:
I agree with this, let's tighten it up into an `include?`

Reply:
ok, Thanks, I will do the change. :rocket: